src/main/java/cn/nukkit/entity/mob/EntityMob.java

```diff
@@ -20,12 +20,7 @@ public EntityMob(FullChunk chunk, CompoundTag nbt) {
     @Override
     public boolean onInteract(Player player, Item item, Vector3 clickedPos) {
         if (item.getId() == Item.NAME_TAG) {
-            if (item.hasCustomName()) {
-                this.setNameTag(item.getCustomName());
-                this.setNameTagVisible(true);
-                player.getInventory().removeItem(item);
-                return true;
-            }
```

Comment:
I didn't know this code was there, I thought it wasn't implemented because the `ItemNameTag` didn't even exist lol. It's still a good PR through because it removed a duplicated code

Reply:
Note: The code wasn't there hehe, it was added by this PR, I messed up :)